inc/languages/english/admin/config_attachment_types.lang.php

```diff
@@ -6,7 +6,7 @@
  */
 
 $l['attachment_types'] = "Attachment Types";
-$l['attachment_types_desc'] = "Here you can create and manage attachment types which define which types of files users can attach to posts.";
+$l['attachment_types_desc'] = "Here you can create and manage attachment types which define which types of files users can attach to posts. Note: Disabling an attachment type will remove all existing attachments of this type from the forum!";
```

Comment:
Looks like such attachments are made inaccessible, rather than deleted, and can be made available again when matching type is restored.

Reply:
So what is your suggestion?

Reply:
How about:
```diff
- will remove all existing attachments of this type from the forum!
+ will make existing attachments of this type inaccessible.
```

Reply:
OK, better. Fixed